doc/source/parameters/dem/lagrangian_physical_properties.rst

```diff
@@ -102,6 +115,18 @@ In this subsection, gravitational acceleration, and the physical properties of t
 
 * The ``Hamaker constant particles`` defines the Hamaker constant of particles for each type. This parameter is used with the DMT force model.
 
+* The ``thermal conductivity particles`` defines the thermal conductivity of particles for each type. This parameter is used in multiphysic DEM, which is not yet supported.
+
+* The ``specific heat particles`` defines the specific heat of particles for each type. This parameter is used in multiphysic DEM, which is not yet supported.
```

Comment:
```suggestion
* The ``specific heat particles`` defines the specific heat of particles for each type. This parameter is used in multiphysic DEM simulations, which are not yet supported.
```

Reply:
It feels redundant to say that for every parameter. Why not remove the statement and instead and a .. warning:: that says (The following DEM parameters are used for multphysics DEM simulations and are currently not supported : )